src/loki/height_action.cc

```diff
@@ -53,7 +53,9 @@ std::vector<PointLL> loki_worker_t::init_height(Api& request) {
       }
       // re-encode it for display if they sent it encoded
       if (options.has_encoded_polyline()) {
-        options.set_encoded_polyline(midgard::encode(shape));
+        // Default to 6 digit precision unless polyline5 is specified
+        int precision = options.shape_format() == polyline5 ? 1e5 : 1e6;
```

Comment:
technically it can also be geojson format too right? i guess we could word the docs to say if its poly5 we use that otherwise we expect poly6. to remove the possibility that someone wants geojson to work.

Reply:
i mean i know the parameter is "encoded_polyline" but i just wanted to mention geojson is a valid shapeformat which we dont support here

Reply:
added a note to the comment here. The API doc for elevation states that polyline5 or polyline6 are the possible options.